app/backend/src/tests/test_api.py

```diff
@@ -489,21 +488,6 @@ def test_ListMutualFriends(db):
         assert mutual_friends[0].user_id == user3.id
 
 
-def test_mutual_friends_from_user_proto_message(db):
-    user1, token1 = generate_user()
-    user2, token2 = generate_user()
-    user3, token3 = generate_user()
-    user4, token4 = generate_user()
-    make_friends(user1, user2)
-    make_friends(user1, user3)
-    make_friends(user4, user2)
-    make_friends(user4, user3)
-
-    with api_session(token1) as api:
-        user4_from_db = api.GetUser(api_pb2.GetUserReq(user=user4.username))
```

Comment:
Just change this to `ListMutualFriends`?

Reply:
There's other tests for ListMutualFriends. I wrote this test specifically to test getting it from the user protobuff message